kernel/procfs.c

```diff
@@ -97,6 +101,25 @@ int procfs_pid_setup(pid_t pid)
         ECHECK(myst_mkdirhier(locals->tmp_path, 777));
     }
 
+    /* Create /proc/sys/kernel directory */
+    {
+        ECHECK(myst_snprintf(
+            locals->tmp_path, sizeof(locals->tmp_path), "/proc/sys/kernel"));
+
+        ECHECK(myst_mkdirhier(locals->tmp_path, 777));
+
+        /* Create /proc/sys/kernel/pid_max entry */
+        ECHECK(myst_snprintf(
```

Comment:
You don't need the string formatting, you could directly use the string literal in `myst_create_virtual_file`.